spec/rspec/matchers/built_in/raise_error_spec.rb

```diff
@@ -290,9 +290,22 @@ def to_s
   end
 end
 
-RSpec.describe "expect { ... }.not_to raise_error(message)" do
+RSpec.describe "expect { ... }.not_to raise_error('message')" do
   it "issues a warning" do
-    expect_warning_with_call_site __FILE__, __LINE__+1, /risks false positives/
+    expect_warning_with_call_site __FILE__, __LINE__+1, /not_to raise_error\(message\)` risks false positives/
+    expect { raise 'blarg' }.not_to raise_error("blah")
```

Comment:
Just nitpicking over the use of single and double quotes on the same line.

Reply:
Whoops! I'm used to doubles, but can adapt either way. I must have copied the local style for the first string and missed it on the second one -.-